doorstop/core/vcs/test/test_commands.py

```diff
@@ -79,44 +85,7 @@ def test_commit(self, mock_call):
 
 
 @patch('subprocess.call')  # pylint: disable=R0904
-class TestMockVCS(BaseTestCase):
-    """Tests for the placeholder VCS plugin."""
```

Comment:
Were these tests not working? I wouldn't expect the new logic to affect the "placeholder VCS" because it never actually calls out to the shell.